content/en/docs/user-guides/how-to/directory-structure.md

```diff
@@ -147,16 +154,28 @@ For testing purpose, the following directory structure is suggested.
 
 Since `signingkeys.json` takes references in absolute paths, it is not required to copy the private keys and certificates used for signing to the above directory structure.
 
+### Caches
+
+Since Notation v1.3.0, CRL files are cached to optimize the network traffic and improve perfromance. The path of cached CRLs follows the pattern below.
```

Comment:
```suggestion
Since Notation v1.3.0, CRL files are cached to optimize the network traffic and improve performance. The path of cached CRLs follows the pattern below.
```